### ISSUE TITLE
Unable to download Colour Contrast Analyser?

### ISSUE THREAD

**Issue by valdudley** [ThePacielloGroup/CCAe#187]
Greetings, I am wanting to download the Colour Contrast Analyser for Windows but when I click to do so I get a list of 12 Assets? Not sure which one to download. https://github.com/ThePacielloGroup/CCAe/releases/tag/v3.1.1

Thanks for helping with this, Valerie Dudley

**Comment by poorgeek** [comment]
For Windows, you likely will want to use the EXE file; https://github.com/ThePacielloGroup/CCAe/releases/download/v3.1.1/CCA-Setup-3.1.1.exe.

**Comment by EleanorMee** [comment]
I'd also love more guidance around what all the assets are and which one you should download for different purposes. The colour contrast analyser is a super useful tool, but working out how to download and install it is not at all easy. It may be obvious to tech geeks, but most bamboozling to the average potential user! Not complaining; just suggesting an improvement to a great tool!

**Comment by ferllings** [comment]
Yes, I agree and there is an action item for that
https://github.com/ThePacielloGroup/CCAe/issues/150

**Comment by ankittiwari2001** [comment]
Hey how to install it locally in ubantu ?
